packages/website/ts/components/documentTitle.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
```

Comment:
This file needs to be snake_case. This convention was consistent before but the b&b guys did not follow it for the 0x.org redesign.